goose/recipes/update-centos-package.yaml

```diff
@@ -48,34 +61,48 @@ parameters:
+  and then create a commit with the changes. Follow exactly these steps:
+  
   1. Find the location of the {{ package }} package at {{ git_url }}.  Always use the {{ git_branch }} branch.
 
-  2. Check if there is a newer version of the {{ package }} available.  That includes detecting the current version of the package and the upstream version.
-    * The current version of the package can be found in the Version field of the RPM .spec file.
-    * The latest version of the package can be found in the upstream project of {{ package }}.  Usually, there is a reference to the upstream project inside the .spec file.  That can be the URL field, a comment or references in the changelog.  Prefer references to Git repositories (e.g., GitHub or Gitlab, etc.) over ordinary websites.
-    * If there is no new version available, print a message and exit.  If there is a new version available follow the instructions below.
-    * Do not clone any repository for detecting the versions.
+  2. Check if the {{ package }} was not already updated to version {{ version }}.  That means comparing the current version and provided version.
+    * The current version of the package can be found in the 'Version' field of the RPM .spec file.
+    * If there is nothing to update, print a message and exit. Otherwise follow the instructions below.
+    * Do not clone any repository for detecting the version in .spec file.
 
   3. Create a local Git repository by following these steps:
     * Create a fork of the {{ package }} package using the glab tool.
     * Clone the fork into the temp directory.
-
+  
   4. Update the {{ package }} to the newer version:
-    * Create a new branch indicating the update of the package and that it was created by {{ git_user }}.
-    * Update the local package.
-    * Only perform changes relevant for the version update.
-    * Use the rpmlint tool to make sure that changes to the .spec file did not introduce new errors.
-    * Build the local .spec file to make sure all changes are correct.
-    * Do not rename variables or comment existing lines out.
-    * Make sure that the commit looks similar to previous ones in the Git log that update the package version.
-    * Do not change if-else branches.
+    * Create a new Git branch named `automated-packaging-update-{{ version }}`.
+    * Update the local package by:
+      * Updating the 'Version' and 'Release' fields in the .spec file as needed, following packaging documentation.
+      * Updating macros related to update (e.g., 'commit') if present and necessary; examine the file's history to see how updates are typically done.
+      * Creating a changelog entry, referencing the Jira issue as "Resolves {{ jira_issue }}".
+      * Downloading sources using `spectool -g -S {{ package }}.spec` and uploading the new sources using `centpkg --release {{ git_branch }} new-sources`.
```

Comment:
If I get it right, `git_branch` = RHEL release, right?

Reply:
`dist_git_branch` sounds good, that's what the `--release` argument stands for really.